Interface/Harp.SyringePump/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Bonsai;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: XmlNamespacePrefix("clr-namespace:Harp.SyringePump", "syr")]
```

Comment:
Since the term `pump` is more used in the firmware than `syringe` I was wondering whether we should use `"pump"` here instead of `"syr"`.